static/app/views/dashboardsV2/widgetBuilder/buildSteps/sortByStep/index.tsx

```diff
@@ -149,6 +149,17 @@ export function SortByStep({
                 sortDirection === SortDirection.HIGH_TO_LOW ? `-${sortBy}` : sortBy;
               onSortByChange(newOrderBy);
             }}
+            tags={tags}
+            filterPrimaryOptions={option => {
+              if (option.value.kind === FieldValueKind.FUNCTION) {
+                return true;
+              }
+
+              return (
+                columnSet.has(option.value.meta.name) ||
+                option.value.meta.name === CUSTOM_EQUATION_VALUE
+              );
+            }}
```

Comment:
I think this is causing equations from getting dropped from the the sort options.
ie, if your y-axis looks like this:
![image](https://user-images.githubusercontent.com/83961295/167172708-84ad38df-8505-4161-8e13-8d34f2a0a3fd.png)

`count() + count()` doesn't appear in the sort options, which it used to do. Not sure if this is intended though, since the user can kind of work around by using `custom equation`.

Reply:
Might also need to make a change in `sortBySelectors.tsx`. I think the equation is contained in `sortByOptions`, but it might not be getting used in one of the `if` blocks

Reply:
https://github.com/getsentry/sentry/pull/34011/commits/5eedac84347366bdd946b6004813ce2d8f440e52

:tada:

Reply:
👍  tested it and the equation is now correctly appearing in the order by

I did notice that the default value for the order by is appearing as empty, which causes the chart to error:
![image](https://user-images.githubusercontent.com/83961295/167222601-bffd0d48-ef7b-4964-88aa-42eb970c9d56.png)

I can leave an approve for this PR in case you want to fix this in another PR